### DISCUSSION TITLE
Is it possible to add extra attributes to the Dataspace configuration that will be included in the DAT?

### DISCUSSION THREAD

**Discussion by vsiop** [International-Data-Spaces-Association/DataspaceConnector#513] [Q&A]
Ηow can we configure the dataspace connector to add extra attributes that will be included in the DAT of Dataspace connector? In particular, we want to add to our dataspace connector  the extendedGuarantee attribute. We want to add the attribute  via the configuration of the connector and not in the DAPS

**Comment by ghost** [comment]
Hello and thank you for the request!
No that is unfortunately not possible, there is no setting in the DSC to add additional claims to the later issued DAT (as far as I understood the request correctly).
